src/Aws/Sqs/Resources/sqs-2012-11-05.php

```diff
@@ -98,6 +98,10 @@
                     'type' => 'string',
                     'location' => 'aws.query',
                 ),
+                'MessageGroupId' => array(
```

Comment:
Adding to last comment, I'm confused since this is not a member for the operation either? Could you tell me more about how you find adding this helpful?